### PR TITLE
Themes documentation

### PR DESCRIPTION
**Pull request by simeonoff**

### CODE REVIEW

**Comment by zdrawku** [comment]
Next week we can rename the description to remove avatar mentions

File: en/components/themes.md [hunk 1]
```diff
@@ -1 +1,149 @@
...
+#### Themes
+
+The second concept is the one of themes. Palettes, wouldn't do much good if they weren't used by a theme. So we have themes for each component, and a global one, that styles the entire application and every component in it. You simply pass your generated palette to the global theme, we take care of the rest. You can, of course, style each component individually to your liking. We will take a closer look at how to do that later in this article.
+
+#### Typography
+
+The last concept revolves around typography. Although we have a default typeface choice, we really want to give you the power to style your application in every single way. Typography is such an important part of that. We provide a method for changing the font family, the sizes and weights for headings, subheadings and paragraph texts in your app.
+
+> [!NOTE]
+> Themeing **requires** [**Sass**](https://github.com/sass/node-sass).
```
Comment (ig-ealbert) [inline en/components/themes.md]: Theming

File: en/components/themes.md [hunk 2]
```diff
@@ -1 +1,149 @@
...
...so many colors in each sub-palette you may be wondering, how on Earth will I know which one is which, right? It's quite simple, really. Each of the colors in the sub-palette has a number. We assign the number `500` to the original color. The lighter shades start from `100` and range to `400`. The darker shades start from `600` and range to `900`. The accent colors have string names `A100`, `A200`, `A400`, and `A700`, respectively. Okay, but now that's only 26 out of 72. Don't worry, there's another sub-palette we give you. One that consists of gray 'colors', called `grays`. It's just like the other two color sub-palettes, but doesn't include any accent variations. Good, now we're up to 26 + 9 for a total of 35 colors. That is still a long way from 74. Where do the other 39 colors come from? Let's solve the final puzzle. Remember you can also have 4 additonional colors for `info`, `success`, `warn` and `error`. So that leaves another 35 colors unaccounted for. Remember the count for the `primary`, `secondary`, and `grays` sub-palettes was exactly 35? The other 35 remaining colors are contrast text colors for each of the colors in the `primary`, `secondary`, and `grays` sub-palettes.
```
Comment (ig-ealbert) [inline en/components/themes.md]: additional

File: en/components/themes.md [hunk 3]
```diff
@@ -1 +1,149 @@
...
+#### Typography
+
+The last concept revolves around typography. Although we have a default typeface choice, we really want to give you the power to style your application in every single way. Typography is such an important part of that. We provide a method for changing the font family, the sizes and weights for headings, subheadings and paragraph texts in your app.
+
+> [!NOTE]
+> Themeing **requires** [**Sass**](https://github.com/sass/node-sass).
+
+### Generating Color Palettes
+
+Our theming library is based on Sass. If you used the **<a href="https://github.com/IgniteUI/igniteui-cli" target="_blank">Ignite UI CLI</a>** to bootstrap your app, you can specify the style extentsion in the **angular-cli.json** config to _scss_, the CLI will take care of compiling the Sass styles for you. If you haven't used Ignite UI CLI then you have to configure your builder to compile Sass styles for you.
```
Comment (ig-ealbert) [inline en/components/themes.md]: extension

**Pull request merged**